### PR TITLE
Add labels explaining the system roles (fate#317481 comment#4)

### PR DESCRIPTION
**Pull request by cwh42**
- https://trello.com/c/5dfbvsIF
- https://bugzilla.suse.com/show_bug.cgi?id=989711

roles

### CODE REVIEW

File: control/control.SLES.xml [hunk 1]
```diff
@@ -303,21 +302,24 @@ defined by a role can be overridden in the next steps if necessary.&lt;/p&gt;</l
           <label>Default System</label>
         </normal_role>
         <normal_role_description>
-          <label></label>
+          <label>• GNOME environment, with BTRFS root (/) partition
+• Separate /home partition (xfs) for disks larger than 20GB</label>
```
Comment (mvidner) [inline control/control.SLES.xml]: It seems the preferred spellings are Btrfs and XFS:

- https://en.wikipedia.org/wiki/Btrfs
- https://en.wikipedia.org/wiki/XFS

File: package/skelcd-control-SLES.changes [hunk 2]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Tue Jul 26 14:05:24 UTC 2016 - cwh@suse.com
+
+- Add labels explaining the system roles (fate#317481 comment#4)
```
Comment (mvidner) [inline package/skelcd-control-SLES.changes]: actually comment 44, but I think we should reference the P1 bug instead: ([bsc#989711](https://bugzilla.suse.com/show_bug.cgi?id=989711))

**Comment by mvidner** [comment]
Nice trick with the bullet :) Unicode 👍

**Comment by kobliha** [comment]
Yep, nice :)! Thx

**Comment by cwh42** [comment]
Changed capitalization of XFS and Btrfs and referenced the bug id instead of the fate comment.

**Comment by mvidner** [comment]
LGTM, thank you!

**Comment by mvidner** [comment]
Hmm, we forgot about the text mode. The unicode bullets are rendered as inverted question marks: https://openqa.suse.de/tests/515687#step/system_role/1

**Pull request merged**